frontend/lib/src/components/widgets/CameraInput/CameraInput.tsx

```diff
@@ -276,7 +273,21 @@ class CameraInput extends React.PureComponent<Props, State> {
     }
   }
 
-  private createWidgetValue(): FileUploaderStateProto | undefined {
+  public componentDidMount(): void {
+    const newWidgetValue = this.createWidgetValue()
+    const { element, widgetMgr } = this.props
+
+    // Set the state value on mount, to avoid triggering an extra rerun after
+    // the first rerun.
+    const prevWidgetValue = widgetMgr.getFileUploaderStateValue(element)
```

Comment:
Just confirming - we want to use the `getFileUploaderStateValue` and `setFileUploaderStateValue` functions instead of a camera input equivalent?
Think a comment to clarify this is intentional would be helpful 😃

Reply:
Comment added, thanks!